WindowsLib/WindowMenu.cpp

```diff
@@ -36,7 +36,7 @@ namespace WinLib {
         {
             auto uiText = icux::toUistring(text);
             menuItemInfo.fMask = MIIM_STRING;
-            menuItemInfo.dwTypeData = (LPWSTR)uiText.c_str();
+            WideCharToMultiByte(CP_ACP, 0, (LPWSTR)uiText.c_str(), -1, menuItemInfo.dwTypeData, NULL, NULL, NULL);
```

Comment:
The solution here is to remove the cast entirely and just have the line be...
```menuItemInfo.dwTypeData = uiText.c_str();```

If compiling in ANSI/multibyte mode it's wrong to provide a LPWSTR for this parameter (though it would be fine compiling in UNICODE mode); uistrings adjust for the compile mode automatically.